docker-compose.test.yml

```diff
@@ -166,48 +168,42 @@ services:
   # available versions for CUDNN_VERSION and NCCL_VERSION_OVERRIDE can be found at
   #   https://developer.download.nvidia.com/compute/cuda/repos/{OS}/x86_64/
 
-  # we need CUDA 10.0 as tensorflow-gpu==1.15.5 is compiled against and linked to CUDA 10.0
-  # torch==1.8.1 is the latest we can test in this setup
-  # mxnet-cu100==1.7.0 does not contain mkldnn headers, and there is no 1.7.0.postx that would have them
-  # there is no mxnet-1.6.0.post0 and mxnet-1.6.0 does not work with horovod
-  # https://github.com/apache/incubator-mxnet/issues/16193
-  # so we test with mxnet 1.5.1
-  test-gpu-gloo-py3_7-tf1_15_5-keras2_2_4-torch1_8_1-mxnet1_5_1_p0-pyspark3_4_0:
+  # Mainline tensorflow-gpu==1.15.5 is compiled against and linked to CUDA 10.0, but appropriate containers aren't
+  # available anymore. Hence, we use the updated Python 3.8 wheel provided by Nvidia, see
+  # https://github.com/NVIDIA/tensorflow.
+  test-gpu-gloo-py3_8-tf1_15_5-keras2_2_4-torch1_12_1-mxnet1_8_0_p0-pyspark3_4_0:
```

Comment:
We also have to deviate from CPU versions for torch and mxnet.

Reply:
thanks, updated